content/03.methods.md

```diff
@@ -1,9 +1,23 @@
-#Materials and Methods
+# Materials and Methods
 
-##Dataset
-Talk about dataset - Pubtator
-Talk about preprocessing Pubtator
-Talk about hand annotations for each realtion
+## Dataset
+The dataset used for this project is called Pubtator [@doi:10.1093/nar/gkt441], which is an open source repository that contains pre-tagged medline abstracts.
+It was downloaded on June 30, 2017 and contained 10,775,748 different abstracts.
+After obtaining these abstracts, we filtered out mention tags that were not contained in hetionet.
+Following the tag filtering step, we used the Stanford CoreNLP parser [@doi:10.3115/v1/P14-5010] to do part of speech (POS) tagging and dependency tree generation.
+Treating sentences with only a single mention as negative, we extracted sentences with two or more mentions, termed candidate sentences. 
+After obtaining candidate sentences, we did a stratified sort of each candiadte into a training set, tuning set and a testing set (shown in Table 1).
+Each sentence was sorted based on the present mention pair.
```

Comment:
I see that this goes with above. Best to work it in there, but as written I also don't quite get it. You stratified sorted by entity or by pairs of entities?